### ISSUE TITLE
Change mode ring is not operational

### ISSUE THREAD

**Issue by phanesaleg** [denniedegroot/com.ring#98]
Hello, I inserted the mode change for ring, but it does not work
![Capture2](https://user-images.githubusercontent.com/108235014/175832133-20660514-40f3-4a47-b380-27c50dc358a0.JPG)

**Comment by Loooogical** [comment]
Yeah, not working for me either. This is the one functionality I desperately want for my system, so that I can activate the Ring alarm automatically with Homey when I leave and comes back to the house. Not sure if this has ever been operational, but if it has been, i truly hope it is fixed soon!

**Comment by Loooogical** [comment]
Never mind, turns out this function is not working for me because I have the alarm base station setup. Guess it can only change modes when there is no alarm added to your Ring system
